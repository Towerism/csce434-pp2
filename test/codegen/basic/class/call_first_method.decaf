class Asdf {
      int print_one() {
          Print(1);
      }
      int print_two() {
          Print(2);
      }
      int print_three() {
          Print(3);
      }
      int print_four() {
          Print(4);
      }
      int print_five() {
          Print(5);
      }
}

int main() {
    Asdf b;
    b = New(Asdf);
    b.print_five();
}