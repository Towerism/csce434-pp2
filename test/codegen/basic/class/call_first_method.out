Loaded: /usr/share/spim/exceptions.s
5